docker/Dockerfile

```diff
@@ -6,20 +9,22 @@ COPY docker/install_rust.sh .
 RUN bash ./install_rust.sh
 
 # Install dev tools
-RUN apt-get update && apt-get install -y valgrind
```

Comment:
Removed the `apt update` and that is already executed inside the `install_rust.sh` script.